OpenRA.Editor/Form1.cs

```diff
@@ -238,7 +238,9 @@ void PrepareMapResources(ModData modData, Map map)
 					if (rsi != null && rsi.EditorPalette != null && rsi.EditorPalette.Contains("terrain"))
 						templatePalette = palette;
 
-					var template = RenderUtils.RenderActor(info, tileset, templatePalette);
+					var race = Program.Rules.Actors["world"].Traits.WithInterface<CountryInfo>().First().Race;
```

Comment:
The code in this file is specific to the legacy editor.  The ingame editor uses the ActorPreview code, which supports race images.  The only missing piece is the properties/init panel in the editor, which is needed for other important functionality such as setting the facing, health, cargo, etc.